phpunit.el

```diff
@@ -239,6 +247,23 @@ https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.anno
   (let ((default-directory (phpunit-get-root-directory)))
     (compile (phpunit-get-compile-command args))))
 
+(defun phpunit--hide-compilation-buffer-if-all-tests-pass (buffer status)
+  "Hide the compilation BUFFER if all tests pass.
+The STATUS describes how the compilation process finished."
+  (with-current-buffer buffer
+    (let* ((buffer-string (buffer-substring-no-properties
+                           (point-min) (point-max)))
+           (buffer-lines (s-lines buffer-string)))
+      (when-let ((ok-msg (car (cl-remove-if-not
```

Comment:
`if-let` and `when-let` are added in Emacs 25. Perhaps we should continue to support Emacs 24.3 a little more.